src/sass/components/ValidationProgress.scss

```diff
@@ -1,33 +1,92 @@
+@keyframes progressIndicator {
+  0% {
+    opacity: 0.4;
+    box-shadow: none;
+  }
+  50% {
+    opacity: 1;
+    box-shadow: 0 1px 2px -1px $color-primary;
+  }
+  100% {
+    opacity: 0.4;
+    box-shadow: none;
+  }
+}
+
 .ValidationProgress {
-  border-top: 1px solid $color-gray-lightest;
-  margin-top: 1em;
   padding-top: 1em;
   .usa-button {
     margin-top: .5em;
   }
-  .usa-unstyled-list>li {
-    margin-bottom: .5em;
+
+  .progressTotal {
+    background-color: $color-gray-lightest;
+    width: 100%;
+    height: 0.2em;
+    &::before {
+      content: "0";
```

Comment:
😃  I think to be consistent with what we're showing it makes sense. I have a few thoughts in my head for some improvements too but need to play some.